src/main/java/frc/robot/subsystems/Vision.java

```diff
@@ -123,13 +126,39 @@ public Optional<EstimatedRobotPose> getEstimatedGlobalPose(Pose2d prevEstimatedR
         return m_photonPoseEstimator.update();
     }
 
-    // private Optional<Pose2d> getTagPose(int tagID){
-    //     // This can fail if the tagID is not part of the map. 
-    //     // Need to check before converting to Pose2d
-    //     return Optional.of(m_aprilTagFieldLayout.getTagPose(tagID).get().toPose2d());
-    // }
+    public Optional<Pose2d> getCentralTagPose() { // gets target closets to center of camera
+        var targetResult = m_aprilTagCamera.getLatestResult();
+        int targetID = -1;
+        if (targetResult.hasTargets()) {
+            // Get lists of targets
+            List<PhotonTrackedTarget> targets = targetResult.getTargets();
+            // make a temp holder var for least Y translation, set to first tags translation
+            //Translation2d translationHolder = targets.get(0).getBestCameraToTarget().getTranslation().toTranslation2d();
+            double minTranslation = targets.get(0).getBestCameraToTarget().getTranslation().toTranslation2d().getY();
```

Comment:
While this will work, you are repeating the same check as the first in the loop. It is much faster to just set the initial value to a really large number, like "1000000.0", and let the first iteration do the work.